src/platform/symbols.map

```diff
@@ -193,4 +193,11 @@ MIRPLATFORM_2.3 {
     mir::graphics::LinuxDmaBufUnstable::?LinuxDmaBufUnstable*;
     mir::graphics::LinuxDmaBufUnstable::buffer_from_resource*;
   };
-} MIRPLATFORM_2.2;
\ No newline at end of file
+} MIRPLATFORM_2.2;
+
+MIRPLATFORM_2.4 {
+ global:
+  extern "C++" {
+    mir::options::x11_scale_opt;
+  };
+} MIRPLATFORM_2.3;
```

Comment:
I don't think we need a new stanza as Mir2.3 has yet to be branched.

Reply:
Ack